### PR TITLE
Tarea 2

### PR DESCRIPTION
**Pull request by AngelMrles7**

### CODE REVIEW

**Comment by emolinaab** [comment]
@AngelMrles7 , remove files outside tarea-2 folder

File: angel-morales/tarea-2/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <header class="content-title" title="titular de noticia">
+        <h3 class="subtitle">Futbol de Alto Nivel</h3>
+        <h1 class="title">
+          La joya <span class="color-title">Colombiana</span>
+        </h1>
+      </header>
+
+      <article class="content-page">
+        <aside class="aside-left">
+          <div class="text-content-left">
```
Comment (emolinaab) [inline angel-morales/tarea-2/index.html:21]: @AngelMrles7 this div should be a section

File: angel-morales/tarea-2/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <p class="text-content">
+              Golazo de Luis Díaz enloqueció a los hinchas del Liverpool
+            </p>
+          </div>
+          <div class="text-content-left">
```
Comment (emolinaab) [inline angel-morales/tarea-2/index.html:26]: @AngelMrles7 this div should be a section

**Review by emolinaab — CHANGES_REQUESTED** [review]

File: angel-morales/tarea-2/index.html (hunk 2 above)
Comment (AngelMrles7) [reply angel-morales/tarea-2/index.html]: Good morning.
I have already made the changes

**Comment by emolinaab** [comment]
@AngelMrles7 , remove files outside tarea-2 folder

**Review by emolinaab — APPROVED** [review]

**Pull request merged**